code/modules/mob/living/blood.dm

```diff
@@ -6,55 +6,166 @@
 				BLOOD SYSTEM
 ****************************************************/
 
+/// Returns whether this mob can have blood.
+/// Use the CAN_HAVE_BLOOD(mob) macro instead.
+/// Unless you're the one updating blood status.
+/mob/living/proc/can_have_blood()
+	return get_default_blood_volume() > 0
+
+/mob/living/carbon/can_have_blood()
+	return !HAS_TRAIT(src, TRAIT_NOBLOOD)
+
+/// Returns the default blood volume of this mob. Useful for healing bloodloss.
+/mob/living/proc/get_default_blood_volume()
+	return initial(blood_volume)
+
+/mob/living/carbon/get_default_blood_volume()
+	return BLOOD_VOLUME_NORMAL
```

Comment:
We should probably just have a `base_blood_volume` var

Reply:
Fair point. In this case a var should be plenty. It's easy enough to refactor back into a proc if need be.